### DISCUSSION TITLE
Graph in terminal

### DISCUSSION THREAD

**Discussion by crmabs** [hpjansson/chafa#230] [Show and tell]
I just bumped into this captured file. Rather crappy, I know. I also know that if I'm not sharing it now then I'll never will. 
The graph in the terminal is a chafa + generated svg using the data getting collected in those other terminals.
Many thanks again for your efforts!

https://github.com/user-attachments/assets/993185ec-e26e-4ebf-8ae4-5a7e47eb72f3

**Comment by hpjansson** [comment]
Nice! Finally proof you can use it to hack into someone's ghost without voiding the warranty (there is no warranty).

(Seriously though, processing something fun? Looks like an interesting use case.)

**Reply by crmabs** [reply]
An Espressif Esp32 Cpu has several free memory counters. They got periodically pushed to a webserver wich collects the data and draws the graph lines with different colours and saves it to an svg file. This is happening in the consoles with only the corners are visible. The third one is a chafa in watch mode drawing the graph. Roughly.
I was running the stuff for a rather long time and I was only interested in the long trends. Namely if there would have been a memory leak we'd be seeing one of the lines getting closer to zero. As you can see they are chaning constantly but they stay within a range. After a few unusable numeric representation of the data this was born. :))))
